subsys/dfu/dfu_target/Kconfig

```diff
@@ -79,6 +79,16 @@ config DFU_TARGET_FULL_MODEM
 	  Enable support for full updates to the modem firmware.
 	  Note that this requires an external flash.
 
+config DFU_TARGET_FULL_MODEM_EXT_FLASH_SIZE
+	hex "Flash size required for storing full modem image"
+	depends on DFU_TARGET_FULL_MODEM
+	default 0x400000 if SOC_NRF9160
+
+config DFU_TARGET_FULL_MODEM_USE_EXT_PARTITION
+	depends on DFU_TARGET_FULL_MODEM
+	select PM_PARTITION_REGION_FMFU_STORAGE_EXTERNAL
```

Comment:
Where is this option defined?

Reply:
https://github.com/nrfconnect/sdk-nrf/blob/main/subsys/partition_manager/Kconfig.template.partition_region